assets/sass/06-components/navigation.scss

```diff
@@ -375,6 +375,7 @@
 
 	// Show top-level sub-menu indicators above mobile-breakpoint-only
 	.menu-item-has-children {
+		margin-right: 0 !important;
```

Comment:
It is best practice to avoid using !important.

I suggest using specificity instead, and placing

`			&.menu-item-has-children {
				margin-right: 0;
			}`

Inside this code starting on line 283:
```
.primary-menu-container > ul > .menu-item {
		@include media(mobile) {
			....
		}
	}
```

(Github removes the formatting here, follow the coding standards :) )